mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+42af412
```

Comment:
never commit changes to this file, it is autogenerated.

Reply:
I've made a note, thanks!

Reply:
See the note about a post-commit hook in the developers' handbook

Reply:
well hidden in https://musescore.org/en/handbook/developers-handbook/compilation/compile-instructions-windows-mingw-git#Source-Code

Reply:
Found it - all set for next PR.

Reply:
You need to get it out of this one